package.json

```diff
@@ -76,20 +75,23 @@
     "vitest": "^0.34.3"
   },
   "dependencies": {
-    "@nextcloud/auth": "^2.1.0",
+    "@nextcloud/auth": "^2.2.1",
     "@nextcloud/axios": "^2.4.0",
-    "@nextcloud/dialogs": "^4.2.0-beta.4",
-    "@nextcloud/files": "^3.0.0-beta.21",
+    "@nextcloud/dialogs": "^5.0.0-beta.6",
+    "@nextcloud/files": "^3.0.0-beta.26",
     "@nextcloud/l10n": "^2.2.0",
-    "@nextcloud/logger": "^2.5.0",
-    "@nextcloud/router": "^2.1.2",
-    "@nextcloud/vue": "^8.0.0-beta.3",
+    "@nextcloud/logger": "^2.7.0",
+    "@nextcloud/router": "^2.2.0",
+    "@nextcloud/vue": "^8.0.0-beta || ^8.0.0",
```

Comment:
I would remove this from deps as already in peerDependencies.

Reply:
After years and lots of reading, I still don't know if peerDeps are exclusive of the other depepdency fields or if you need to have it in both :see_no_evil: :grin: 

I'm fine removing it, but it also doesn't seem to create issues as is. Do you know more and/or have ressources for me to read? :grin:

Reply:
AFAIK the peer dependency takes precedence, so there should be no difference
https://github.com/npm/rfcs/issues/324
https://github.com/npm/arborist/commit/b0997c91083f1ebd1c4e8d4e996b7aa5e4c39ad2